libs/explorers/comparison-tools/src/lib/help-links/help-links.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="help-links">
```

Comment:
This component encapsulates all the help links (Legend and Visualization Overview).  The link for Legend works but the Visualization Overview link will be implemented in [MG-133](https://sagebionetworks.jira.com/browse/MG-133)